backend/functions/scorecard_parser.py

```
@@ -270,41 +244,41 @@ def parse_scorecard_rows(self, line, scorecard_data) -> dict:
 
         Returns: dict: Updated scorecard data.
         """
-        # Read elements from html
-        raw_label = line.find_element(By.CSS_SELECTOR, 'div.line-left > p, div.line-left > p.body-bold')\
-            .text.strip()
+        # Get row label from the sticky first cell
+        try:
+            raw_label = line.find_element(By.CSS_SELECTOR, 'div.sticky span.truncate').text.strip()
+        except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)